src/hooks/use-session.js

```diff
@@ -1,16 +1,14 @@
 import {useState, useEffect} from 'react'
-import {supabase} from '../utils/supabase-client'
 
-export default function useSession () {
+export default function useSession (sessionProvider) {
```

Comment:
how about naming it `database` to stay consistent with the `DbSession` context?

Reply:
it is not really `database`, as it is `supabase` we are passing.
Supabase in this case , is the "database provider" and "session provider" (as in auth0, and other session as a service providers).

We could rename it to database, but it seems to me we are not interested in making a database query here,
but a "auth" query.